addons/website/static/src/scss/website.scss

```diff
@@ -1122,6 +1122,33 @@ table.table_desc tr td {
     }
 }
 
+// Navbar templates
+.o_navbar_showcase{
+    .navbar-brand{
+        margin: auto;
+    }
+    .navbar-nav{
+        margin-top: 10px;
+    }
+}
+
+.o_navbar_brand{
+    .container{
+        -webkit-flex-wrap: wrap !important;
+        flex-wrap: wrap !important;
+    }
+    .navbar-collapse{
+        flex-basis: 100%;
+    }
+    .navbar-brand, .navbar-nav{
+        margin: auto;
+    }
+    .navbar-brand{
+        margin-top: 10px;
+        margin-bottom: 10px;
```

Comment:
There is probably a meaningful bootstrap variable (or an expression using those) to use instead of `10px` (and even when hardcoding values, most of them should be expressed in rem units instead of px.